timely_beliefs/beliefs/utils.py

```diff
@@ -694,19 +689,39 @@ def interpret_special_read_cases(
     elif len(df.columns) == 3:
         # datetimes in 1st and 2nd column, and value in 3rd column
         df.columns = ["event_start", "belief_time", "event_value"]
-        df["event_start"] = pd.to_datetime(df["event_start"])
-        if df["event_start"].dt.tz is None:
-            df["event_start"] = df["event_start"].dt.tz_localize(
-                timezone, ambiguous="infer"
+        df["event_start"] = to_sensor_timezone(df["event_start"], sensor, timezone)
+        df["belief_time"] = to_sensor_timezone(df["belief_time"], sensor, timezone)
+    return df
+
+
+def to_sensor_timezone(
+    s: pd.Series, sensor: "classes.Sensor", timezone: Optional[str]
+) -> pd.Series:
+    """Convert the timezone of the series to the timezone of the sensor.
+
+    In case the series contains naive datetimes, they are first localized using the 'timezone' parameter.
+
+    :param s:           series with datetime representations
+    :param sensor:      object with a timezone property to which all datetimes are converted
```

Comment:
Actually, now I realize we only need the sensor param for it's timezone.
Clearer: use two parameters called expected_timezone and alternative_timezone? Or other better names. ..

Reply:
I guess I'll go with `timezone_to_convert_to` and `timezone_to_localize_to`.